content/en/virtual-private-cloud.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Virtual Private Cloud
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+
+A virtual private cloud (VPC) is a secure, isolated private cloud hosted within a public cloud. It allows users to do everything they are able to do on an ordinary cloud, but the private cloud is hosted remotely by a public cloud provider. It gives user the ability to define and control a virtual network that is isolated from all other public cloud customers, creating a private, secure place on the public cloud.
+
+## Problem it addresses
+
+There are often Security Breaches on the data hosted on public clouds due to not regular monitoring.<br>
+Outdated Software leads to performance issues. <br>
+At times, organizations don't exactly know how much capacity they will need, and end up overbuying to ensure they don’t come up short and end up paying more space so that they don't fall short.<br> 
+Users also face issues with Sync, Integration and Deployment.<br>
+There is always a need to buy more hardware to extend support. Thus increasing the carbon consumption.
+
+## How it helps
+
+In a Virtual Private Cloud, only authenticated user can get access to the information. Moreover, VPCs are monitored for security by Cloud Security Experts regularly.<br>
```

Comment:
Speculative and irrelevant.

Reply:
Hey @JasonMorgan! It would be really great if you could help me with where I am getting wrong.
Best,
Akshat